bot/discord/commands/votemute.js

```diff
@@ -0,0 +1,64 @@
+const ms = require('ms');
+const Discord = require('discord.js');
+const db = require('quick.db');
+const mutesData = new db.table("muteData");
+let mutes = {};
+
+exports.run = async(client, message, args) => {
+  let modlog = message.guild.channels.cache.find(channel => channel.id === config.DiscordBot.modLogs);
+  if (!message.guild.me.hasPermission('MANAGE_ROLES')) return message.reply('Sorry, i dont have the perms to do this cmd i need MANAGE_ROLES. :x:');
+  if (message.member.roles.cache.find(r => r.id === "898041754564046869") == null) return message.reply("sorry, but it looks like you're too much of a boomer to run this command.");
```

Comment:
A vote mute command.. where you got to donate? shouldn't it be anyone?

Reply:
can be only started by a donator, can be reacted by anyone else

Reply:
it shouldn't only be allowed to be started by donators

Reply:
there's no role other than donator that fits it perfectly? An untrusted member can team up and basically mute someone